npm/util/osutil_linux/current-azure-chains_test.go

```diff
@@ -0,0 +1,142 @@
+//go:build !windows
+// +build !windows
+
+package linuxutil
+
+import (
+	"testing"
+
+	"github.com/Azure/azure-container-networking/common"
+	testutils "github.com/Azure/azure-container-networking/test/utils"
+	"github.com/stretchr/testify/require"
+)
+
+const grepOutputAzureChainsWithPolicies = `Chain AZURE-NPM (1 references)
+Chain AZURE-NPM-ACCEPT (1 references)
+Chain AZURE-NPM-EGRESS (1 references)
+Chain AZURE-NPM-EGRESS-123456 (1 references)
+Chain AZURE-NPM-INGRESS (1 references)
+Chain AZURE-NPM-INGRESS-123456 (1 references)
+Chain AZURE-NPM-INGRESS-ALLOW-MARK (1 references)
+`
+
+var listAllCommandStrings = []string{"iptables", "-w", "60", "-t", "filter", "-n", "-L"}
```

Comment:
Seems like a useful command. You could export it. Maybe in some follow up PRs we move test helpers into their own pkg?

Reply:
perhaps in another PR to clean up UTs? There are a couple areas of improvement